engines/misc/src/main/java/org/alfresco/transform/misc/transformers/TextToPdfContentTransformer.java

```diff
@@ -78,19 +91,29 @@ public class TextToPdfContentTransformer implements CustomTransformerFileAdaptor
     private static final byte BB = (byte) 0xBB;
     private static final byte BF = (byte) 0xBF;
 
+    @Value("${transform.core.misc.pdfBox.defaultFont}")
+    String pdfBoxDefaultFont;
 
     private final PagedTextToPDF transformer;
 
+    protected static final int DEFAULT_FONT_SIZE = 10;

```

Comment:
Can't this be private? Also, I'd move it up next to the other `private static final` constants of this class as a best practice.

Reply:
Changed `DEFAULT_FONT_SIZE` visibility to `private` and moved its location up next to the other constants.